template.tpl

```diff
@@ -191,13 +214,19 @@ ___TEMPLATE_PARAMETERS___
         "paramName": "tagType",
         "paramValue": "googleAdsEvent",
         "type": "EQUALS"
+      },
+      {
+        "paramName": "tagType",
+        "paramValue": "googleAdsCallConversionsEvent",
+        "type": "EQUALS"
       }
     ]
   },
   {
     "type": "TEXT",
     "name": "googleAdsConversionId",
     "displayName": "Conversion ID (optional)",
+    "help": "This is needed only if the Conversion ID differs from the one configured in the Freshpaint Destination",
```

Comment:
I see that conversion ID is optional here, but I also see it being used in `processGoogleAdsCallConversionsEvent`. How would we get that information if the customer doesn't provide it in gtm?

Reply:
The googleAdsConversionId field you're referring to is used only by the 'Google Ads' destination.

There is a separate Conversion ID field, googleAdsCallConversionsConversionId, used only by the 'Google Ads Call Conversions' action.

Reply:
Ah I misread stuff. Thanks!